content/en/docs/contribute/localization.md

```diff
@@ -73,7 +73,9 @@ For an example of adding a label, see the PR for adding the [Italian language la
 
 ### Find community
 
-Let Kubernetes SIG Docs know you're interested in creating a localization! Join the [SIG Docs Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/). Other localization teams are happy to help you get started and answer any questions you have.
+Let Kubernetes SIG Docs know you're interested in creating a localization! Join the [SIG Docs Slack channel](https://kubernetes.slack.com/messages/C1J0BPD2M/) and the [SIG Docs Localization Slack channel](https://kubernetes.slack.com/archives/C0191RDKHU1). Other localization teams are happy to help you get started and answer any questions you have.
```

Comment:
```suggestion
Let Kubernetes SIG Docs know you're interested in creating a localization! Join the [SIG Docs Slack channel](https://kubernetes.slack.com/messages/sig-docs) and the [SIG Docs Localizations Slack channel](https://kubernetes.slack.com/messages/sig-docs-localizations). Other localization teams are happy to help you get started and answer any questions you have.
```

Reply:
Great Suggestions!,  I will send up a new commit we these changes!